Test/PsvTsvTest.cs

```diff
@@ -25,6 +25,14 @@ public static void ReadOGlycoPsms()
             Assert.AreEqual(9, parsedPsms.Count);
         }
 
+        [Test]
+        public static void ReadExcelEditedPsms()
```

Comment:
you could add a test for one of the fields that has quotes and show that it was parsed correctly

Reply:
New version of the test ensures that fragment ions (the field most often enclosed in quotes) are parsed correctly